modules/channel-vonage/src/backend/client.ts

```diff
@@ -278,6 +279,7 @@ export class VonageClient {
   }
 
   private async sendCarousel(event: sdk.IO.Event, payload: any) {
+    let counter = 0
```

Comment:
I know I'm being picky right now but I'm not a fan of having a counter polluting the code, can you change the logic to use a classic for loop instead of a for of, thus making the counter declaration and assignment on a single line.

Using Promise.mapSeries could be a tiny bit more elegant as well.

Reply:
I agree with you, good idea. I'll make the changes.